raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -372,27 +371,23 @@ private void maybeFireLeaderChange() {
 
     @Override
     public void initialize() {
-        try {
-            quorum.initialize(new OffsetAndEpoch(log.endOffset().offset, log.lastFetchedEpoch()));
+        quorum.initialize(new OffsetAndEpoch(log.endOffset().offset, log.lastFetchedEpoch()));
 
-            long currentTimeMs = time.milliseconds();
-            if (quorum.isLeader()) {
-                throw new IllegalStateException("Voter cannot initialize as a Leader");
-            } else if (quorum.isCandidate()) {
-                onBecomeCandidate(currentTimeMs);
-            } else if (quorum.isFollower()) {
-                onBecomeFollower(currentTimeMs);
-            }
+        long currentTimeMs = time.milliseconds();
+        if (quorum.isLeader()) {
+            throw new IllegalStateException("Voter cannot initialize as a Leader");
+        } else if (quorum.isCandidate()) {
+            onBecomeCandidate(currentTimeMs);
+        } else if (quorum.isFollower()) {
+            onBecomeFollower(currentTimeMs);
+        }
 
-            // When there is only a single voter, become candidate immediately
-            if (quorum.isVoter()
+        // When there is only a single voter, become candidate immediately
+        if (quorum.isVoter()
                 && quorum.remoteVoters().isEmpty()
                 && !quorum.isCandidate()) {
```

Comment:
indent is not correct here.

Reply:
Newly added code, indentation seems to be correct?

Reply:
No, you can see the change, you move this line `if (quorum.isVoter()` left (4 spaces I guess), so the following 2 lines should also move left.

Reply:
Thank you very much for your precious time. I understand it and I will fix it